github/PullRequest.py

```diff
@@ -775,6 +775,19 @@ def merge(self, commit_message=github.GithubObject.NotSet, commit_title=github.G
             self.url + "/merge",
             input=post_parameters
         )
+
+
+        if delete_branch == True:
+
+            # Check for other pull requests referencing the same ref.
+            # If found, abort.
+            remaining_pulls = self.head.repo.get_pulls(head=self.head.ref)
+            if remaining_pulls > 0:
```

Comment:
```suggestion
            if remaining_pulls:
```
I guess this will work already